core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -19,10 +19,10 @@ package kafka.server
 
 import java.util.Properties
 
+import DynamicConfigs.Broker._
```

Comment:
In the diff below, you are using a fully qualified name (i.e. `DynamicConfigs.Client.*`) in a few places and it seemed like you avoided that in `BrokerConfigHandler`. So, it seemed like the intent was to avoid the FQN in classes that only handled a particular type of dynamic config. The way to achieve that in Scala is to have the import within the relevant scope. But maybe I read too much into the difference between usages.